Astro 3/1/23 (#1942)
astronomer/docs: astro/release-notes.md

**Latest CLI Version**: 1.11.0 ([Release notes](cli/release-notes.md))

## February 28, 2023

### Bug fixes

> - To protect Astro's monitoring functionality, you can no longer override the values of the following environment variables:

Reviewer: ```suggestion
- To ensure the reliability of Astro monitoring, you can no longer override the values of the following environment variables:
```

  
    - `AIRFLOW__METRICS__STATSD_ON` 
    - `AIRFLOW__METRICS__STATSD_HOST`
    - `AIRFLOW__METRICS__STATSD_PORT`
    - `AIRFLOW__METRICS__STATSD_ALLOW_LIST` 
    - `AIRFLOW__METRICS__STATSD_STATSD_CUSTOM_CLIENT_PATH`
    - `AIRFLOW__METRICS__STATSD_PREFIX`

    You can still set new values for these variables, but those values are automatically overwritten in the Astro data plane. See [Platform variables](platform-variables.md).
  

> - Fixed an issue where a user could be provisioned multiple accounts if they logged in using differently cased characters in their login email address.

Reviewer: ```suggestion
- Fixed an issue where a user could provision multiple accounts when their login email address included differently cased characters.
```

**Latest CLI Version**: 1.11.0 ([Release notes](cli/release-notes.md))

> ## February 28, 2023

Author: ```suggestion
## March 1, 2023
```

### Access Astronomer through the Okta dashboard

> If your Organization uses Okta as your Astro identity provider, you can now log in to Astro directly from your [Okta apps dashboard](https://help.okta.com/eu/en-us/Content/Topics/end-user/dashboard-overview.htm). If you're already authenticated through Okta, you no longer have to reauthenticate in Astro if you access it through your dashboard.

Reviewer: ```suggestion
If your Organization uses Okta as your Astro identity provider, you can now log in to Astro directly from your [Okta Apps dashboard](https://help.okta.com/eu/en-us/Content/Topics/end-user/dashboard-overview.htm). If you've already been authenticated by Okta, you no longer have to be authenticated by Astro when you access it through your dashboard.
```

If you want users to always authenticate through Astro, you can disable this feature. See [Restrict identity provider initiated logins](manage-organization.md#restrict-identity-provider-initiated-logins).

### Additional improvements

> - Ingress to GCP and Azure clusters is now limited to control plane IPs. This change will be implemented on all clusters in the coming weeks.

Reviewer: ```suggestion
- Access to Google Cloud Platform (GCP) and Azure clusters is now limited to control plane IPs. This change will be implemented on all clusters in the coming weeks.
```

### Bug fixes

- To protect the functionality of Astro monitoring services, you can no longer override the values of the following environment variables:
  
    - `AIRFLOW__METRICS__STATSD_ON` 
    - `AIRFLOW__METRICS__STATSD_HOST`
    - `AIRFLOW__METRICS__STATSD_PORT`
    - `AIRFLOW__METRICS__STATSD_ALLOW_LIST` 
    - `AIRFLOW__METRICS__STATSD_STATSD_CUSTOM_CLIENT_PATH`
    - `AIRFLOW__METRICS__STATSD_PREFIX`

>     You can still set new values for these variables, but those values are automatically overwritten in the Astro data plane. See [Platform variables](platform-variables.md).

Author: ```suggestion
    You can still set new values for these variables, but the values will be automatically overwritten in the Astro data plane. See [Platform variables](platform-variables.md).
```

### Additional improvements

> - Ingress to AWS clusters is now limited to control plane IPs. This change will be implemented on all clusters in the coming weeks.

Author: ```suggestion
- Ingress to the Airflow UI and API on Astro clusters is now limited to control plane IPs. This change will be implemented on all clusters in the coming weeks.
```
